environments/qa/modules/localconfig/manifests/nodes.pp

```diff
@@ -1,10 +1,5 @@
-node 'qa0' {
-    $nodetype = 'qa'
+node 'dev0' {
```

Comment:
I have re-used the same nodetype definition for local, dev and qa, there is no need for a separate definition for now as they are all pretty similar. Admittedly it is confusing that they are all called "dev0".